tests/api2/test_007_pool.py

```diff
@@ -192,13 +232,15 @@ def test_12_import_nonascii_msdosfs():
     expect_state(job_id, "SUCCESS")
 
 
-def test_13_look_if_Каталог_slash_Файл():
+def test_15_look_if_Каталог_slash_Файл(request):
```

Comment:
What is this testing exactly?

Reply:
I am not sure I did not add this test.

Reply:
I move it out of the pool test and move it in pool_import and mark it to skip all pool import test on HA.